MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,15 +20,46 @@ final class FeedViewModel {
 	
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onFeedLoadError: Observer<String>?
 	
 	func loadFeed() {
 		onLoadingStateChange?(true)
 		feedLoader.load { [weak self] result in
-			if let feed = try? result.get() {
-				self?.onFeedLoad?(feed)
+			guard let self = self else { return }
+			switch result {
+			case .success(_):
+				if let feed = try? result.get() {
+					self.onFeedLoad?(feed)
+				}
```

Comment:
Since the `result` is unwrapped in the `.success` case, you can get the feed directly, without `result.get`

```suggestion
			case let .success(feed):
				self.onFeedLoad?(feed)
```